FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/test/test_freertos_tcp.c

```diff
@@ -93,7 +93,7 @@ TEST( Full_FREERTOS_TCP, prvParseDnsResponse )
         0x23, 0x00, 0x04, 0x22, 0xd7, 0xe6, 0xa4, 0xc0, 0x72, 0x00, 0x01, 0x00, 0x01, 0x00, 0x00, 0x00,
         0x23, 0x00, 0x04, 0x36, 0x95, 0x5e, 0x45
     };
-    const uint32_t ulExpectedAddress = 0xdb41d322;
+    const uint32_t ulExpectedAddress = 0xf0f5d622;
```

Comment:
what is this for? how did this address appeared ?

Reply:
This had to be changed as how the DNS addresses are cached changed. See this PR by Gary here: https://github.com/aws/amazon-freertos/pull/1981.
Also, the PR for changing the same thing in AFR: https://github.com/aws/amazon-freertos/pull/2044